### ISSUE TITLE
The animation effect will be different

### ISSUE THREAD

**Issue by wkz2003** [baronha/ting#33]
For example, the animation of a checkmark is sometimes displayed from bottom to top like handwriting, while other times a complete checkmark is displayed first and then the animation is played. I think both are good, but the effect is a bit inconsistent not very good.
My mobile platform is Android
```
  toast({
    title: 'success',
    position: 'bottom',
    duration: 1,
    message: 'xxx',
  });
```
Another issue I found that I'm not sure if it's a feature or a bug is that the duration of the animation can only be an integer, something like 1.5 doesn't work

**Comment by mrevanzak** [comment]
on android, duration value convert to int so it doesnt work with float i think. i dont use kotlin so just my guess

**Comment by 4cc3ssX** [comment]
This issue should be fixed in this PR #38🚀
